### PR TITLE
dependency: gopkg.in/fsnotify.v1 -> github.com/fsnotify/fsnotify

### PR DESCRIPTION
**Pull request by grooverdan**
Change as per notice here: https://github.com/go-fsnotify/fsnotify

github.com/fsnotify/fsnotify was already in github.com/fsnotify/fsnotify

### CODE REVIEW

File: topology/probes/netns/netns.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ import (
 	"syscall"
 	"time"
 
-	"gopkg.in/fsnotify.v1"
+	"github.com/fsnotify/fsnotify"
```
Comment (nplanel) [inline topology/probes/netns/netns.go]: should be "gopkg.in/fsnotify/fsnotify.v1" as gopkg.in/fsnotify point to the old name

File: vendor/vendor.json [hunk 2]
```diff
@@ -2436,12 +2436,6 @@
 			"revision": "442357a80af5c6bf9b6d51ae791a39c3421004f3",
 			"revisionTime": "2016-12-22T12:58:16Z"
 		},
-		{
```
Comment (nplanel) [inline vendor/vendor.json]: please update govendor via:

govendor fetch gopkg.in/fsnotify/fsnotify.v1

**Comment by nplanel** [comment]
Thanks for the report
As our CI is up again and need a fixup, I amend your commit to keep you as the author in PR #886 
I will close this PR

**Comment by grooverdan** [comment]
cheers.

**Pull request closed without merging**